### PR TITLE
Be a date: matcher for providing datetime type check

### PR DESCRIPTION
**Pull request by jsmolina**
Including be_a_date matcher + unit test.

### CODE REVIEW

File: pyshould/matchers.py [hunk 1]
```diff
@@ -365,6 +366,12 @@ def describe_to(self, desc):
         desc.append_text('a class')
 
 
+class IsDate(TypeMatcher):
+    """ Check if the value is a date """
+    types = datetime
```
Comment (drslump) [inline pyshould/matchers.py]: this should be `datetime.date`:

```py
>>> from datetime import date
>>> from datetime import datetime
>>> isinstance(datetime.now(), datetime)
True
>>> isinstance(date.today(), datetime)
False
>>> isinstance(datetime.now(), date)
True
>>> isinstance(date.today(), date)
True
```

**Comment by drslump** [comment]
thanks @jsmolina! I left a comment regarding the comparison since I think it's best to also match `datetime.date` instances with this matcher. Could you review it?

**Comment by jsmolina** [comment]
great idea @drslump !

**Comment by drslump** [comment]
thanks @jsmolina! -- will issue a new minor release to reflect the changes shortly

**Pull request merged**